jsk_kinova_robot/jsk_kinova_description/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>jsk_kinova_description</name>
+  <version>1.1.0</version>
+  <description>The jsk_kinova_description package</description>
```

Comment:
please add more description on why we need this package.

Reply:
Added